src/components/dashboardHeader/dashboardHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from 'react-router-dom'
+import paths from '../../routing/paths'
+import styles from './dashboardHeader.module.css'
+
+const DashboardHeader = () => (
+  <header className={styles.root}>
+    <div className={styles.bar}>
+      <span className={styles.headerContainer}>
+        <h1 className={styles.header}>
+          <Link className={styles.headerLinkLarge} to={paths.dashboard.main}>
+            Skyrim Inventory
+            <br className={styles.bp} /> Management
+          </Link>
+          <Link className={styles.headerLinkSmall} to={paths.dashboard.main}>
+            S. I. M.
+          </Link>
```

Comment:
I considered using CSS to set the content here instead of showing and hiding two different `Link` components. However, that would've involved giving up control over where the line break happens in "Skyrim Inventory Management" - without the `<br>` element, it breaks after "Skyrim" and that doesn't look good.